### DISCUSSION TITLE
GraphQL integration plugin

### DISCUSSION THREAD

**Discussion by marcelo-earth** [nasa/openmct#5817] [Show and tell]
👋  Hi OpenMCT community!

I create [Graphem](http://github.com/360macky/graphem), a plugin that connects historical and realtime data from a [GraphQL server](https://graphql.org) to visualize telemetry in Open MCT.

The purpose I did this for is to learn more about the Open MCT API.

The [first project where I used it](http://github.com/360macky/simetral) was to display data from the **Artemis I** mission with [Horizon System](https://ssd.jpl.nasa.gov/horizons/). As the data is for a September 27 release, and the mission was rescheduled, I will update it when there is a new date and information.

I sent an email to NASA Open MCT team so this plugin can be published in the list of community plugins. I hope to see it there soon!

**Edit:** This plugin has an [official webpage](https://graphem.space).

**Comment by jvigliotta** [comment]
We have a PR up for getting your plugin in the community plugins list: https://nasa.github.io/openmct/plugins/, should be up soon! Thanks for submitting it!

**Reply by marcelo-earth** [reply]
Awesome. ☺️ Thank you so much!

**Reply by marcelo-earth** [reply]
Hi @jvigliotta! 😊 Quick question. Is the PR in a private NASA repository?
